cmd/kyverno/main.go

```diff
@@ -76,10 +76,6 @@ func main() {
 	flag.BoolVar(&fqdncn, "fqdn-as-cn", false, "use FQDN as Common Name in CSR")
```

Comment:
can we remove this flag

Reply:
This is introduced for EKS cluster, will need to verify..

Reply:
We removed that logic - it was for older releases.

Reply:
Seems like it's used here: https://github.com/kyverno/kyverno/blob/35aa3149c85dc609289efa774db9f3451eef4823/pkg/tls/tls.go#L122-L125

Are you saying this is no longer needed?

Reply:
Yes, its been fixed in EKS 1.16. We can remove.

Reply:
As of now, Kyverno still supports 1.14 at minimum, should we remove it when we drop 1.15?

Reply:
The issue was for CSRs sent to the EKS control plane. We no longer do that, since we generate self-signed CA and certs. I think we can safely remove.